### PR TITLE
[Interactive Graph Locked Figures] Add controls to move a whole locked polygon

### PR DESCRIPTION
**Pull request by nishasy**
## Summary:
Right now, if someone wants to move an entire locked polygon, they’d have to
manually change every single point’s coordinate pair.

I added some arrow buttons to accomplish this.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2115

## Test plan:
`yarn jest packages/perseus-editor/src/components/__tests__/locked-polygon-settings.test.tsx`

Storybook
- http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-figures-current
- Open the locked polygon settings
- Play around with the buttons and confirm that the polygon moves accordingly

## Screenshots & video:

NOTE: The arrows might look like they're too far to the right, but it doesn't look like this
in prod where the width of the settings is shortened to 306px by webapp styles.

Screenshot 2024-08-14 at 4 14 34 PM

https://github.com/user-attachments/assets/87b6a085-24a2-4085-90d4-6c0e11b97de3

### CODE REVIEW

File: packages/perseus-editor/src/components/graph-locked-figures/locked-polygon-settings.tsx
Comment (aemandine) [inline packages/perseus-editor/src/components/graph-locked-figures/locked-polygon-settings.tsx:61]: [Non-blocking, newbie question]
This is a cool way to specify what string values the function should accept! Is this strategy preferred over an enum? (Maybe for the sake of simplicity, since it's only used once?) Thanks:)

**Review by aemandine — APPROVED** [review]
Hi Nisha! This is my first code review, so I would perhaps wait for someone else's  approval as well. That said, I paired with Third for this, and we think it looks great! I left two comments, one of which is just a question because I'm still learning TypeScript style. Thanks!

File: packages/perseus-editor/src/components/__tests__/locked-polygon-settings.test.tsx
Comment (nishasy) [reply packages/perseus-editor/src/components/__tests__/locked-polygon-settings.test.tsx:222]: Oops! Yes 😅 I see I forgot that below as well.

File: packages/perseus-editor/src/components/graph-locked-figures/locked-polygon-settings.tsx
Comment (nishasy) [reply packages/perseus-editor/src/components/graph-locked-figures/locked-polygon-settings.tsx:61]: As far as I can tell, enums and union types like this pretty much serve the same function. I do it like this since it's more consistent with the rest of the codebase. I'd be curious to hear what other people on the Perseus team think though.

**Pull request merged**
